docs/romanisim/catalog.rst

```diff
@@ -31,6 +31,36 @@ Following these required fields is a series of columns giving the fluxes of the
+The simulator API includes a few simple tools to generate parametric distributions of stars and galaxies.  
+
+The ``make_stars`` and ``make_galaxies`` routines make random catalogs of stars and galaxies.  The number of stars and galaxies can be adjusted.  Likewise, the power law index by which the sources' magnitudes are sampled can be adjusted, as can their limiting magnitudes.  Galaxy Sersic parameters, half-light radii, and position angles are chosen at random, with a rough attempt to make brighter galaxies appropriately larger (i.e., conserving surface brightness).  Stars can be chosen to be distributed with a King profile.  This functionality is however very rudimentary and limited, and is better suited for toy problems than real scientific work.  
+
+We expect scientific uses to be driven by custom-created catalogs rather than the simple routines above, and provide the ``make_cosmos_galaxies`` and ``make_gaia_stars`` routines outlined below to provide that funcionality. 
+
+
+Using COSMOS Galaxies and GAIA Stars
+====================================
+
+.. figure:: ./cosmos_gaia.png
+  :width: 600
+
+  Level 3 Mosaic of cosmos and gaia sources with radius = 0.1 degree (search area visible)
+
+
+The simulator can utilize the COSMOS catalog to generate sources based off of real galaxies via the ``make_cosmos_galaxies`` method. The COSMOS survey is a 2 square degree collection of over 2 million galaxies with multi-wavelength photometry (Weaver et al. 2022)[https://arxiv.org/pdf/2110.13923]. A streamlined catalog for the simulator is provided with the code, but for those so interested, the latest full catalog can be found at https://cosmos.astro.caltech.edu/ and used with this method. 
+ 
+The simulator performs the following operations on the catalog sources:
+
+* Only galaxies that have "ultra-deep" Ultra Vista wavelength measurements are selected from the COSMOS catalog. This trims the catalog to 0.62 square degrees, and to a peak magnitude depth of ~25. 
+* The source density of the COSMOS catalog as well as the input viewing area are used to determine the number of sources simulated.
+* Roman bands are approximated by linear interpolations of nearest HST and Ultra Vista bands.
+* Galaxies are placed in random positions about the sky.
+* While galaxy shapes are preserved, they are given randomly chosen concentrations (1 <= n <= 4).
+* Sources are randomly oriented.
+* Source fluxes are lightly perturbed.
+
+The simulator allows for passage of a user-specified COSMOS file (one which must have the same tabular format as COSMOS2020_CLASSIC_R1_v2.2_p3.fits) via the ``filename`` keyword. The catalog area can be set with ``cat_area``.
+
+The simulator can also generate the appropriate false-color stars for the image from the GAIA catalog via the ``make_gaia_stars`` method, in place of the random stars in ``make_stars``.
```

Comment:
false-color to me implies some attempt to get the star colors right, but this follows gaia.gaia2romanisimcat that just does AB color = 0 in all bands.

```suggestion
The simulator can also generate place Gaia stars at the appropriate locations using the ``make_gaia_stars`` method, in place of the random stars in ``make_stars``.  Note that these stars have fluxes exactly equal to the Gaia G band fluxes in all bands; i.e., no attempt is made to predict accurate colors of stars.
```

Reply:
Changed